### DISCUSSION TITLE
set default for cleaning repeats / times

### DISCUSSION THREAD

**Discussion by FusselTV** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#694] [Q&A]
Does anyone know how to set the default selection for cleaning repeats to two.
![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/89834718/ed4ae3f0-ec31-4e5a-aa3b-668a0c95f12a)

**Comment by PiotrMachowski** [comment]
It's not possible at this moment. You can hardcode it if you want to

**Reply by FusselTV** [reply]
Good to know, thank you for the quick reply. I'll make a feature request. It's just a nice-to-have for lazy ppl.
